CHANGELOG.md

```diff
@@ -1,6 +1,16 @@
 # OHHTTPStubs — CHANGELOG
 
+## [Future Release]
+
+* Updating default Swift Version to 4.2, but keeping code compiling in Swift 2.3, 3.x, and 4.x.  
+  [@jeffctown](https://github.com/jeffctown)
+* Updating project settings for Xcode 10.1.  
+  [@jeffctown](https://github.com/jeffctown)
+* Updating example projects for new version.
+  [@jeffctown](https://github.com/jeffctown)
```

Comment:
Might be nice to also reference all the people who submitted a PR for updating to all those successive Xcode/Swift versions (even though we ended up not merging then because I was so late on my OSS time to merge them but since they made the effort…)

Reply:
Love this idea.  Updated in c9b28c9.